hnn_core/viz.py

```diff
@@ -610,6 +615,8 @@ def plot_tfr_morlet(dpl, freqs, *, n_cycles=7., tmin=None, tmax=None,
         The name of a matplotlib colormap, e.g., 'viridis'. Default: 'inferno'
     colorbar : bool
         If True (default), adjust figure to include colorbar.
+    colorbar_loc: str or None
+        The location of colorbar. Set to right side if None.
```

Comment:
Are the only options `'inner'` and `None`? If that's the case it seems like this should just be a boolean for placing it inside or outside.